releasenotes/notes/remove_answer_generator-e7100f82c1859fcb.yaml

```diff
@@ -0,0 +1,3 @@
+---
+deprecations:
```

Comment:
According to reno convention, this should not appear on `deprecations`.
https://github.com/deepset-ai/haystack/blob/a55024bee7b2b105f9152688e27d1adc0e8cb9e6/releasenotes/config.yaml#L32

So, we should probably compile the `upgrade` section, explaining to users to replace this component with the `PromptNode`; maybe we can link some docs or tutorials.